controller/api/models.py

```diff
@@ -267,11 +292,15 @@ def scale(self, user, structure):  # noqa
                 to_add.append(c)
                 container_num += 1
                 diff -= 1
+
         if changed:
-            if to_add:
-                self._start_containers(to_add)
-            if to_remove:
-                self._destroy_containers(to_remove)
+            if "scale" in dir(self._scheduler):
+                self._scale_containers(scale_types, to_remove)
```

Comment:
Because we need to just update the RC with with the number of replicas and it will create or delete the extra pods for us which is different from the existing implementations.